components/templates/Test.tsx

```diff
@@ -45,6 +45,17 @@ const useStyles = makeStyles(theme => ({
   },
   textColorLightMode: {
   },
+  externalLogsButton: {
+    backgroundColor: theme.palette.secondary.dark,
+    width: "auto",
+    height: "auto",
+    marginLeft: "10px",
+    border: "1px #696969 solid",
+    fontSize: "12px",
+    color: "#7CFC00",
```

Comment:
Juan, do you mind using "green" for consistency of the colours? 😄 Or something less neon/aggressive

Reply:
hahahaha sure @opishcheiko  😄 you can actually ignore this style. Cause the user will be able to pick the color for this type of buttons 

<img width="571" alt="Screenshot 2020-11-25 at 17 02 46" src="https://user-images.githubusercontent.com/14996980/100259489-2c258180-2f40-11eb-9531-717aaff5cb2e.png">

Reply:
ahaha, whyyyyy are you so cruel 😢 , you want to let user destroy the perfect colour palette of our site 👩‍🎨

Reply:
Please feel free to change this palette of colors to a way to like the most 👍